src/plugins/data/public/search/search_source/search_source.ts

```diff
@@ -301,6 +295,9 @@ export class SearchSource {
     this.requestStartHandlers.push(handler);
   }
 
+  /**
+   * get resulting DSL
```

Comment:
I'm thinking sometimes the property names and function names are more descriptive then a short comment like this. For example, `getSearchRequestBody` tells me more than "get resulting DSL". Maybe in this case it is worth expanding the description?

```suggestion
   * Returns body contents of the search request, often referred as query DSL.
```